bin/deploy.sh

```diff
@@ -1,33 +1,34 @@
-#!/bin/bash
-
-set -e
+#!/usr/bin/env bash
 
 ENV=$1
-RELEASE_TAG=$2
+RELEASE_TAG=$(git rev-parse HEAD)
 APP_NAME=badger-brain
 AWS_ACCOUNT=578418881509
 AWS_REGION=eu-west-1
 ECR_REPO=$AWS_ACCOUNT.dkr.ecr.$AWS_REGION.amazonaws.com/$APP_NAME
 EB_BUCKET=elasticbeanstalk-$AWS_REGION-$AWS_ACCOUNT
 
+if [ -z "$ENV" ] || [ -z "$RELEASE_TAG" ]
+then
+  echo Usage:
+  echo "  deploy.sh ENVIRONMENT"
+  exit 1
+fi
+
+set -eu
+set -o pipefail
+
 # Authenticate
 eval $(aws ecr get-login --region=$AWS_REGION)
 
 # Build Docker image
-if [ "$ENV" == "live" ]
-then
-  VERSION=$RELEASE_TAG
-  docker build -t $APP_NAME:$RELEASE_TAG .
-  docker tag $APP_NAME:$RELEASE_TAG $ECR_REPO:$RELEASE_TAG
-  docker push $ECR_REPO:$RELEASE_TAG
-else
-  VERSION=$ENV
-  docker build -t $APP_NAME:$ENV .
-  docker tag $APP_NAME:$ENV $ECR_REPO:$ENV
-  docker push $ECR_REPO:$ENV
-fi
+VERSION=$RELEASE_TAG
+docker build -t $APP_NAME:$RELEASE_TAG .
+docker tag $APP_NAME:$RELEASE_TAG $ECR_REPO:$RELEASE_TAG
+docker push $ECR_REPO:$RELEASE_TAG
 
 # Apply docker image path to Dockerrun.aws.json template
+cp Dockerrun.aws.json.template Dockerrun.aws.json
```

Comment:
Moving the compiled Dockerrun.aws.json out of the repo so the deployer doesn't need to undo the changes to the file after each deploy. Removes risk of accidentally committing the edited version.